thedmstoolkit/character.py

```diff
@@ -0,0 +1,112 @@
+from toolkit.models import Character
+
+
+def Arrange(CharacterID, *StatArray):
```

Comment:
what do the arguments look like for this function?
right now it would have to be called with 6 individual numbers as args
ex. Arrange(character_id, 2, 1, 3, 4, 5, 4, 6)

Reply:
I was hoping to pass it an array of 6 numbers, but am unsure how to do that I may need to just have it pass 6 numbers

Reply:
ah, just put `StatArray`, and it will take an array,
you can even specify the type by writing `StatArray: list[int]` to make it easier